tests/test_testUtils.py

```diff
@@ -574,6 +598,38 @@ def testSkypixHandling(self):
         # PixA dataset should have been retrieved by runTestQuantum
         run.assert_called_once_with(a=data)
 
+    def testLintConnectionsOk(self):
+        lintConnections(VisitConnections)
+        lintConnections(PatchConnections)
+        lintConnections(SkyPixConnections)
+
+    def testLintConnectionsMissingMultiple(self):
+        class BadConnections(PipelineTaskConnections,
+                             dimensions={"tract", "patch", "skymap"}):
+            coadds = connectionTypes.Input(
+                name="coadd_calexp",
+                storageClass="ExposureF",
+                dimensions={"tract", "patch", "band", "skymap"},
+            )
+
+        with self.assertRaises(AssertionError):
+            lintConnections(BadConnections)
+        lintConnections(BadConnections, checkMissingMultiple=False)
+
+    def testLintConnectionsExtraMultiple(self):
+        class BadConnections(PipelineTaskConnections,
+                             dimensions=["tract", "patch", "band", "skymap"]):
```

Comment:
I'm presuming that making this a list instead of a set is what makes this a bad `Connections`, but it's not immediately obvious to me why, so maybe add a brief explanatory comment?

Reply:
It should be irrelevant. I'll add a comment that I'm trying to test the linter's robustness to different collection types.